node/pkg/watchers/sui/watcher.go

```diff
@@ -351,36 +303,31 @@ func (e *Watcher) Run(ctx context.Context) error {
 				return ctx.Err()
 
 			default:
-				_, msg, err := ws.Read(ctx)
-				if err != nil {
-					logger.Error(fmt.Sprintf("ReadMessage: '%s'", err.Error()))
-					p2p.DefaultRegistry.AddErrorCount(vaa.ChainIDSui, 1)
-					suiConnectionErrors.WithLabelValues("channel_read_error").Inc()
-					return err
-				}
-
-				var res SuiEventMsg
-				err = json.Unmarshal(msg, &res)
+				fmt.Println("Getting events...", time.Now().Format(time.RFC3339))
+				// This will return an array of events in the correct range and order
+				event, err := e.getEvents()
```

Comment:
nit: `event` is actually the response wrapper, containing `Data` which is the list of events, right?